File: pset6/cash.py
from cs50 import get_float
from math import floor


def main():
    while True:
        dollars_owed = get_float("Change owed: ")
        cents_owed = floor(dollars_owed * 100)

        if cents_owed > 0:
            break

    quarters = cents_owed // 25
    dimes = (cents_owed % 25) // 10
    nickels = ((cents_owed % 25) % 10) // 5
    pennies = ((cents_owed % 25) % 10) % 5
    total = quarters + dimes + nickels + pennies
    print(total)


if __name__ == "__main__":
    main()

File: pset1/cash.c
#include <math.h>
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    float dollars;
    int counter = 0;
    do
    {
        dollars = get_float("Change : ");
    }while(dollars<=0);
    int cents = round(dollars * 100);
    while (cents >=25){
        counter++;
        cents-=25;
    } 
    while (cents >=10){
        counter++;
        cents -=10;
    } 
    while (cents >= 5){
        counter ++;
        cents -=5;
    } 
    while (cents >= 1){
        counter++;
        cents --;
    }
    printf("%i\n",counter);
}